cmake/liburiparser.pc.in

```diff
@@ -0,0 +1,12 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=@CMAKE_INSTALL_FULL_BINDIR@
+libdir=@CMAKE_INSTALL_FULL_LIBDIR@
+includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
+
+Name: liburiparser
```

Comment:
Let's drop the "lib" from "liburiparser" in `Name: ` and filename please. Maybe it should have been called like that from the beginning, but it's been plain "uriparser" for years now.